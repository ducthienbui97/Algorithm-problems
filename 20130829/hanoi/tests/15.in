3
15
CA AB CB AC BC BA
15
CB BC CA BA AC AB
15
BA BC CA AC CB AB
