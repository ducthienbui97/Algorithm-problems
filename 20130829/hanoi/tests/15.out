32767
4782969
9565937
